deals/pb/deals.proto

```diff
@@ -38,8 +38,9 @@ message StoreRequest {
 }
 
 message StoreReply {
-    repeated string cids = 1;
-    repeated DealConfig failedDeals = 2;
+    string dataCid = 1;
```

Comment:
Now there's an extra return attr with the `dataCid`, will be used for retrieval.